### PR TITLE
Updates the Syndicate revolver into a new revolver (with the same stats)

### PR DESCRIPTION
**Pull request by ReinaCoder**
## About The Pull Request
Makes the syndicate revolver a subtype of the normal revolver and gives it its own sprite, name (S-357) and a new uplink description. 
Pics: 
image
image

Before:
image
## Why It's Good For The Game
Currently the revolver is a bland, generic, brown gun that doesn't look like a sci-fi revolver. Now it has a cleaner, more unique design that fits into the setting and into the Syndicate standard of high tech ballistics.

Changing the uplink description gives players a clearer idea of what they are buying.

Moving the Syndicate .357 from the base of all revolvers into its own entry allows for future changes on the Syndicate revolver to not impact every single revolver in the game.

## Changelog
:cl:
add: The Syndicate uplink revolver has been changed from the default .357 to the S-357. It has the same stats and price as the old version but with a new sprite.
qol: Adds a better explanation on the .357 uplink entry so players have an better understanding of what they are buying.
/:cl:

### CODE REVIEW

**Comment by Serxule** [comment]
doesnt really look that syndicate, and looks more like a laser gun than any other ballistic weapons we have (actually none of them look like this)

**Comment by Farquaar** [comment]
Call me a soulless barbarian, but I like it. The new sprite looks like a modern revolver instead of a cowboy weapon straight out of the American civil war.

**Comment by Serxule** [comment]
> Call me a soulless barbarian, but I like it. The new sprite looks like a modern pistol instead of a cowboy weapon straight out of the American civil war.

it doesnt look bad, it just doesnt fit with being a .357

**Comment by bluetshirtguy** [comment]
ugly future ass lookin modern turd.

**Review by Fikou — CHANGES_REQUESTED** [review]
needs to be a subtype used by the uplink. stuff like gatfruit grows into a .357 and a futuristic design with gamer light does not fit for that (and the sprite is in a different palette anyway)

**Comment by Fikou** [comment]
(personal opinion not maintainer request) the description of the item in the uplink calls it brutally simple. it's just a simple, lethal and iconic weapon on a spess station. i think turning it into a futuristic gun removes the charm

**Comment by Serxule** [comment]
> the description of the item in the uplink calls it brutally simple.

thats a good point too actually

**Comment by GuillaumePrata** [comment]
> it doesnt fit with the general style of other syndicate weapons

You have it the other way around.
All the syndicate ballistics are high tech weaponry, just look at the sprites of the bulldog, C-20r, L6, sniper, even the surplus rifle went from an mosin (another iconic old gun we have) to a SMG.

The station is the one with the old ballistics, just look at each version of the Detective revolver, the double shotgun, the autorifle was the higher tech of ballistics the station had and it still was simple.

**Comment by MTandi** [comment]
I believe PigeonVerde is already planning to update this sprite, along with some other guns. And that one is closer to the original and fits much better.

**Comment by ReinaCoder** [comment]
> needs to be a subtype used by the uplink. stuff like gatfruit grows into a .357 and a futuristic design with gamer light does not fit for that (and the sprite is in a different palette anyway)

this is done

File: code/modules/projectiles/guns/ballistic/revolver.dm [hunk 1]
```diff
@@ -97,6 +97,11 @@
 	if(last_fire && last_fire + 15 SECONDS > world.time)
 		. = span_notice("[user] touches the end of [src] to \the [A], using the residual heat to ignite it in a puff of smoke. What a badass.")
 
+/obj/item/gun/ballistic/revolver/syndicate
+	name = "\improper .357 revolver"
+	desc = "A suspicious revolver. Uses .357 ammo."
```
Comment (Fikou) [inline code/modules/projectiles/guns/ballistic/revolver.dm:102]: if its already set to that by parent no need to set it again

**Comment by itseasytosee** [comment]
I think it looks nice, and lethal. The green decals are the most questionable part of the design.

File: code/modules/projectiles/guns/ballistic/revolver.dm (hunk 1 above)
Comment (ReinaCoder) [reply code/modules/projectiles/guns/ballistic/revolver.dm:102]: I'm gonna change the name and description so they aren't the same as the normal revolver and to add some flavour

**Comment by ReinaCoder** [comment]
Drafting to come up with a new name and description to the revolver to add flavour and so it's not the same as the parent.

> I think it looks nice, and lethal. The green decals are the most questionable part of the design.

Here's how it looks without them 
image

**Comment by calliophis-maculiceps** [comment]
it looks like a chew toy

**Comment by itseasytosee** [comment]
I think it would be better to change the description of the base gun to be non suspicious or whatever. If you do end up changing the description, be sure to make it short and witty.

**Comment by wraith-54321** [comment]
Honestly I think it just needs a slightly longer barrel.

**Comment by Wallemations** [comment]
I think this would look good as a toy, but not as a proper gun. It looks like it's made of plastic.

**Comment by blackdav123** [comment]
put the wood back on :(

also in regards to why the 357 looks so much more outdated compared to other syndicate weaponry it always gave me the impression of "we tried upgrading it but nothing beats deadly simplicity"

**Comment by bluetshirtguy** [comment]
Alright here's a better comment: It lacks anything interesting, your revolver is if ubisoft sprited for ss13, it's a bland modern gun with a green sight. What is possibly wrong with a BIG FAT revolver of the original sprite? what's wrong with the syndicate using some lower tech equipment? you just want SCI FI SCI FI and if something looks old it surely doesn't belong in a world influenced by old 80's sci fi and retro aesthetic.

**Comment by ATH1909** [comment]
the chamber looks a bit... small?

**Comment by itseasytosee** [comment]
Solid description

**Comment by Serxule** [comment]
> > it doesnt fit with the general style of other syndicate weapons
> 
> You have it the other way around. All the syndicate ballistics are high tech weaponry, just look at the sprites of the bulldog, C-20r, L6, sniper, even the surplus rifle went from an mosin (another iconic old gun we have) to a SMG.

you have to be joking, right?
They are simple ballistic guns 
-the m-90gl is literally a p90
-the pp-95 is pretty much just a pp-19 bison, with a green mag
-the cr20 looks a bit strange for combat use but its not very scifi
-the l6 is a classic LMG which looks almost exactly like the m249 just without a stock, not to mention the m249 is called the M249 SAW
-the bulldog looks very military rather than scifi
-the sniper is a pretty classic .50 cal sniper rifle design (the scope is wild now but thats new)
-the ansem is literally just a basic black pistol



Screenshot_2994


The station however, they have multiple laser guns, energy guns, disablers, and a bunch of upgrade kits for laser guns (im not going to bother listing every one of them)

File: code/modules/projectiles/guns/ballistic/revolver.dm [hunk 2]
```diff
@@ -96,7 +96,10 @@
 /obj/item/gun/ballistic/revolver/ignition_effect(atom/A, mob/user)
 	if(last_fire && last_fire + 15 SECONDS > world.time)
 		. = span_notice("[user] touches the end of [src] to \the [A], using the residual heat to ignite it in a puff of smoke. What a badass.")
-
+/obj/item/gun/ballistic/revolver/syndicate
+	name = "\improper S-357"
+	desc = "A 7 round revolver chambered in .357. It has 'Scarborough Arms' engraved on it"
+	icon_state = "syndicate_revolver"
```
Comment (Fikou) [inline code/modules/projectiles/guns/ballistic/revolver.dm:102]:
```suggestion

/obj/item/gun/ballistic/revolver/syndicate
	name = "\improper S-357"
	desc = "A 7 round revolver chambered in .357. It has 'Scarborough Arms' engraved on it"
	icon_state = "syndicate_revolver"

```
why are you removing newlines stinky

**Review by Fikou — CHANGES_REQUESTED** [review]

**Comment by GuillaumePrata** [comment]
> you have to be joking, right?
They are simple ballistic guns

You are misunderstanding what I said, by high tech I mean top of the line ballistics, not sci fi.
A cowboy revolver is a simple ballistics, I wouldn't call a shotgun with a rotating double barrel for custom ammo as simple ballistics.

The ones with simple and cheap ballistics is NT, if we use lore for a second it just makes sense.
The station invests in energy weapons, they have advanced lasers but the ballistics on station are as simple self defense guns by a few crewmembers.
The syndicate invests in ballistic weapons, they have advanced ballistics that have a lot of firepower, attachments, etc.

Get gatfruit .357 and put it together with all the syndie ballistics on that picture, then do the same with all the station ballistics, tell me which group a cowboy revolver fits best.

**Comment by Serxule** [comment]
> You are misunderstanding what I said, by high tech I mean top of the line ballistics, not sci fi. A cowboy revolver is a simple ballistics, I wouldn't call a shotgun with a rotating double barrel for custom ammo as simple ballistics.

a rotating DB isnt very complicated or hard to make (it would literally work the same as a revolver)

> The ones with simple and cheap ballistics is NT, if we use lore for a second it just makes sense. The station invests in energy weapons, they have advanced lasers but the ballistics on station are as simple self defense guns by a few crewmembers. The syndicate invests in ballistic weapons, they have advanced ballistics that have a lot of firepower, attachments, etc.

this is just wrong except for the syndicate part (and even then), NT sells the older ballistics to the station as surplus weapons that are now outdated (mosins being apart of a russian kit which likely isnt even sold by NT, since its restricted behind the broadcast mode on the cargo board), the detective gets the .38 revolver, called the colt detective special, because well, the name explains itself, NT focuses primarily on energy weapons as they're less costly in the long run and dont require (as much) maintenance

> Get gatfruit .357 and put it together with all the syndie ballistics on that picture, then do the same with all the station ballistics, tell me which group a cowboy revolver fits best.

gatfruit doesnt even make sense in the first place (ah yes a plant generates metal, a specific type of ammo, and a fully functional firing mechanism, along with an already existing design of a revolver), it's barely something to stand on for an argument

**Comment by GuillaumePrata** [comment]
> a rotating DB isnt very complicated or hard to make (it would literally work the same as a revolver)

You just ignored half of the Bulldog features there and the one you talked about, while still simple design wise, it still is more advanced than what the station shotguns use.

> this is just wrong except for the syndicate part

Then proceeds to agree with me that NT uses all the outdated ballistics...?

> gatfruit doesnt even make sense in the first place

You're going on a tangent and not answering my point about which group of ballistics the old and new sprite fits better. I called it the gatfruit .357 as to differentiate it from the S-357 this PR adds.

**Comment by Helg2** [comment]
It looks like cursed Rhino revolver, and it sucks.

**Comment by Mothblocks** [comment]
@Fikou Do you still not want the redesign? If so just close it

**Comment by Serxule** [comment]
> if its not used for this save it for something else cause it looks cool on its own. kinda reminds me of a dart/flare gun

i agree, it looks cool as something _other_ than a .357

**Comment by optimumtact** [comment]
I do not think this should have been merged, the original sprite better fits the retro future aesthetic, the new one looks too futuristic and that is wrong for ss13 style

**Comment by dawsonkeyes** [comment]
> I do not think this should have been merged, the original sprite better fits the retro future aesthetic, the new one looks too futuristic and that is wrong for ss13 style

it also doesn’t even fit the aesthetic of existing matte-black guns like the M90, it’s a middleman in both ways and really does not look the part

**Comment by Acer202** [comment]
Looks like a good circuit gun replacement, perhaps? But I don't know about the syndie revolver...

**Comment by Mothblocks** [comment]
but i dont care one way or the other

**Comment by Mothblocks** [comment]
wait the spritetainers didnt even approve this pr. i thought they did. am i fucking stupid

**Comment by LemonInTheDark** [comment]
bucket

**Comment by SmoSmoSmoSmok** [comment]
This is a awesome guns I love this guns it’s another good sprite work by Reina coding 👍 well done on this gusn I can’t waited 2 bought it and killing the players

**Comment by biobootX** [comment]
Unpopular opinion: good resprite with nice choice of colors
tg codebase does not have retrofuturistic style in any form anymore

**Pull request merged**